README.md

```diff
@@ -25,12 +25,25 @@ pnpm add @unleash/nextjs
 This package will attempt to load configuration from
 [Next.js Environment variables](https://nextjs.org/docs/basic-features/environment-variables).
 
-| Prefixable     | Variable                     | Default                                                   | Used in                                              |
-| -------------- | ---------------------------- | --------------------------------------------------------- | ---------------------------------------------------- |
-| `NEXT_PUBLIC_` | `UNLEASH_BASE_URL`           | `http://localhost:4242/api`                               | `FlagProvider`, `getFrontendFlags`                   |
-| **No**         | `UNLEASH_API_TOKEN`          | `default:development.unleash-insecure-api-token`          | `getDefinitions`                                     |
-| `NEXT_PUBLIC_` | `UNLEASH_FRONTEND_API_TOKEN` | `default:development.unleash-insecure-frontend-api-token` | `FlagProvider`, `getFrontendFlags`                   |
-| `NEXT_PUBLIC_` | `UNLEASH_APP_NAME`           | `nextjs`                                                  | `FlagProvider`, `getFrontendFlags`, `getDefinitions` |
+When using Unleash **client-side**, with `<FlagProvider />` or `getFrontendFlags()` configure:
+
+  - `NEXT_PUBLIC_UNLEASH_FRONTEND_API_URL`. URL should end with `/api/frontend` or `/proxy`
+  - `NEXT_PUBLIC_UNLEASH_FRONTEND_API_TOKEN` [client-side Unleash token](https://docs.getunleash.io/reference/api-tokens-and-client-keys#front-end-tokens)
+
+If using **server-side** (SSR, SSG, API), using `getDefinitions()` and `evaluateFlags()`, set:
+
+  - `UNLEASH_SERVER_API_URL` of you instance. URL should end with `/api`
+  - `UNLEASH_SERVER_API_TOKEN` [server-side Unleash client token](https://docs.getunleash.io/reference/api-tokens-and-client-keys#client-tokens)
+
+#### Detailed explanation
+
+| Prefixable     | Variable                     | Default                                                   |
+| -------------- | ---------------------------- | --------------------------------------------------------- |
+| `NEXT_PUBLIC_` | `UNLEASH_SERVER_API_URL`     | `http://localhost:4242/api`                               |
+| `NEXT_PUBLIC_` | `UNLEASH_FRONTEND_API_URL`   | `<(NEXT_PUBLIC_)UNLEASH_SERVER_API_URL>` `/frontend`      |
```

Comment:
What does the default mean here? Does it append `/frontend` to the server API url? Maybe we can express that more clearly by joining the two strings?

```suggestion
| `NEXT_PUBLIC_` | `UNLEASH_FRONTEND_API_URL`   | `<(NEXT_PUBLIC_)UNLEASH_SERVER_API_URL>/frontend`      |
```